man/systemd.service.xml

```diff
@@ -916,12 +916,14 @@
 
     <para>Each command line is split on whitespace, with the first item being the command to
     execute, and the subsequent items being the arguments. Double quotes ("…") and single quotes
-    ('…') may be used, in which case everything until the next matching quote becomes part of the
-    same argument. Quotes themselves are removed. C-style escapes are also supported. The table
-    below contains the list of known escape patterns. Only escape patterns which match the syntax in
-    the table are allowed; other patterns may be added in the future and unknown patterns will
-    result in a warning. In particular, any backslashes should be doubled. Finally, a trailing
-    backslash (<literal>\</literal>) may be used to merge lines.</para>
+    ('…') may be used to wrap a whole item (the opening quote may appear only after whitespace, and
```

Comment:
> the opening quote may appear only after whitespace

That's not quite true. An opening quote may also appear immediately following `ExecXXX=`.
```
ExecStart="/path with spaces/mydaemon" ...
```